src/plopp/backends/pythreejs/point_cloud.py

```diff
@@ -43,6 +44,7 @@ def __init__(self,
         self._x = x
         self._y = y
         self._z = z
+        self._id = uuid.uuid4().hex
```

Comment:
I need the `_id` in the plotly line so that it know how to remove a line (matplotlib artists have a `.remove()` method that just removes them from the axes, but plotly doesn't have that).
I then forsee that giving artists ids will most probably become useful in the future (just like graph nodes already have ids).
But yes, putting ids everywhere isn't technically required at this point. Shall I remove them?